### DISCUSSION TITLE
Is it possible to use gem with Cucumber + Capybara + RSpec?

### DISCUSSION THREAD

**Discussion by gsguma** [donv/capybara-screenshot-diff#71] [Q&A] [answered]
I'm trying to implement this gem using Cucumber + Capybara + RSpec, but it didn't work. Is it possible to use this gem in the cucumber steps?

I tried the following changes:

**env.rb:**
```
require 'capybara/screenshot/diff'

...

RSpec.configure do |config|
  config.include Capybara::Screenshot::Diff
end
```

**test_steps.rb:**
```
And(/^visit google$/) do
  visit 'https://google.com'

  screenshot 'test'
end
```

**Gemfile:**
```
  ...

  gem 'capybara-screenshot-diff'
  gem 'oily_png', platform: :ruby

  ...
```

And I'm getting the error `undefined method 'screenshot' for #<...+Capybara::Screenshot::Diff+Capybara::Screenshot::Os+Capybara::RSpecMatchers+Capybara::DSLRSpecProxyInstaller+Capybara::DSL+RSpec::Matchers+Cucumber::Glue::ProtoWorld+ ...> (NoMethodError)`

If this gem is cucumber compatible, do you have any idea that can help me?

Thank you in advance!

**Comment by pftg** [comment, marked as answer]
Hi @gsguma 

For now, there is no Cucumber integration. I'll check on the weekend and add it.
But I would appreciate your help in verifying it. Can you test my PR on real cases?

Thank you for being so interested in the project!

**Reply by gsguma** [reply]
Hi @pftg,
Sure, it will be my pleasure to "help you help me"!

By the way, congratulations on the project, it's been very useful, as soon as I have more time, I will learn better how it works and I will contribute more.

Thanks for replying/trying! 👍
